hat/assets/js/apps/Iaso/domains/entities/entityTypes/hooks/requests/forms.ts

```diff
@@ -28,11 +28,15 @@ export const useGetForm = (
 
 export const useGetForms = (
     enabled: boolean,
+    fields?: string[] | undefined,
 ): UseQueryResult<Form[], Error> => {
+    let url = '/api/forms/?fields=id,name,latest_form_version';
+    if (fields) {
+        url += `,${fields.join(',')}`;
+    }
     return useSnackQuery({
         queryKey: ['forms'],
-        queryFn: () =>
-            getRequest('/api/forms/?fields=id,name,latest_form_version'),
+        queryFn: () => getRequest(url),
```

Comment:
If you merge the previous suggestion:

```suggestion
        queryFn: () => getRequest(apiUrl),
```

Reply:
I've made the changes you proposed :+1: there it's the `url` we want though